tests/integ/scala/test_dataframe_reader_suite.py

```diff
@@ -344,6 +374,47 @@ def test_read_csv_with_format_type_options(session, mode):
     ]
 
 
+@pytest.mark.parametrize("mode", ["select", "copy"])
+def test_read_csv_with_format_type_options_alias(session, mode):
+    test_file_colon = f"@{tmp_stage_name1}/{test_file_csv_colon}"
+    options = {
+        "sep": "';'",
+        "skip_blank_lines": True,
+        "header": True,
+        "lineSep": "'0x0a'",  # new line needs escape
+    }
+    df1 = (
+        get_reader(session, mode)
+        .schema(user_schema)
+        .options(options)
+        .format("csv")
+        .load(test_file_colon, schema=user_schema, header=True, format="csv", sep=";")
+    )
+    res = df1.collect()
+    res.sort(key=lambda x: x[0])
+    assert res == [Row(1, "one", 1.2), Row(2, "two", 2.2)]
```

Comment:
could you split this test into two part, one without using `kwargs` in `load` and one without adding `options`. Here it is hard to figure out what the test is trying to test for. If it is making sure that `kwargs` get precedence over options, then we should have different values for the options

Reply:
Good suggestion. splittin in 3. With aliases but using standard csv, with kwargs and using options and kwargs to show how kwargs have priority